src/SkillInteractor.ts

```diff
@@ -79,17 +79,20 @@ export abstract class SkillInteractor {
      * Passes in an intent with slots as a simple JSON map: {slot1: "value", slot2: "value2", etc.}
      * @param intentName
      * @param slots
-     * @param callback
      */
-    public intended(intentName: string, slots?: any, requestFilter?: RequestFilter): Promise<any> {
+    public intended(intentName: string, slots?: any): Promise<any> {
         try {
-            return this.callSkillWithIntent(intentName, slots, requestFilter);
+            return this.callSkillWithIntent(intentName, slots);
```

Comment:
in order for the try to catch the error here, you have to do 
`return await this.callSkillWithIntent(intentName, slots);`

Reply:
I added that try/catch for debugging - removed it